### DISCUSSION TITLE
Search results empty on load when Filename:{searchTerms} is used Query Template

### DISCUSSION THREAD

**Discussion by clayjam** [microsoft-search/pnp-modern-search#1959] [Q&A] [answered]
The search results web part doesn't show any results on page load when I add Filename:({searchTerms}) to the 'Query Template'.

Results only show when a user enters a search value.

I would like to initially show results so users can use the Search filter web part without entering any search text

**Comment by BrinaD1** [comment]
What happens if you use Filename:({searchTerms}*) and add the wildcard asterisk to the end of your query? I have used this in the past to get initial results to show.

**Comment by wobba** [comment, marked as answer]
If you want results before a query is entered use the default value option.

![image](https://user-images.githubusercontent.com/51104/160236891-107ace5c-3001-4703-9926-fce2b0bae14c.png)

**Reply by clayjam** [reply]
Many thanks - worked perfectly.
